### ISSUE TITLE
Problem with wifi connection via accesspoint

### ISSUE THREAD

**Issue by felix6262** [HomeSpan/HomeSpan#915]
Hello, 

i implemented the accesspoint before my homespan.begin and the control button to my sketch. now i cant connect to wifi. 
```
homeSpan.setApSSID("HomeSpan-Setup-Kellerbelüftung");
homeSpan.setApPassword("homespan-01");
homeSpan.setApTimeout(300);
homeSpan.enableAutoStartAP();
```

i can connect to the accesspoint via Iphone. but the page to put in the wifi values doesnt show up. 
I tried opening chrome or safari, and tried the following in the adress bar:
captive.apple.com
http://0.0.0.0/
http://1.1.1.1/
msftconnecttest.com
http://neverssl.com/

**Comment by felix6262** [comment]
the following is shown in the serial monitor.  

Accessory ID:      C0:A5:B6:95:9A:59                               LTPK: C4D9FD85F34CF07C3CA8B56F1B7FE7913813611D0522034A1F2E5276CF0B3E2A
12:24:10.829 -> No Paired Controllers
12:24:10.829 -> 
12:24:10.829 -> Accessory configuration number: 1
12:24:10.829 -> 
12:24:10.829 -> *** WIFI CREDENTIALS DATA NOT FOUND.  AUTO-START OF ACCESS POINT ENABLED...
12:24:10.829 -> 
12:24:10.829 -> *** Starting Access Point: HomeSpan-Setup-Kellerbelüftung / homespan-01
12:24:10.876 -> 
12:24:10.876 -> Scanning for Networks...
12:24:10.876 -> 
12:24:17.151 ->   1) FRITZ!Box 7490
12:24:17.151 ->   2) Frittenbude
12:24:17.151 ->   3) MagentaWLAN-LSQK
12:24:17.151 ->   4) WLAN-M5S3Y7
12:24:17.151 -> 
12:24:17.151 -> Ready.
12:24:20.149 -> 
12:24:20.149 -> *** Access Point Terminated.  Restarting...
12:24:20.149 -> 

i also added a controllbutton plus controlLED. The LED is blinking very fast. The only thing i can do with the button is to rest everything. and i accidently started the factory reset, but just by holding the button for about 2 secounds... 

above:

`#define ControlButton 27 `

in setup: 
```
pinMode(ControlButton, INPUT);
homeSpan.setControlPin(ControlButton);
homeSpan.setCommandTimeout(180);
homeSpan.setStatusPin(GreenLED);
homeSpan.setStatusAutoOff(300);
```

**Comment by felix6262** [comment]
okay, i found the problematic line: 

i implemented a Control Button like this
```
#define ControlButton 27 
#define GreenLED 33
```
setup:
```
pinMode(ControlButton, INPUT);
 pinMode(GreenLED, OUTPUT);

```
```
homeSpan.setControlPin(ControlButton);     
  homeSpan.setCommandTimeout(180);           
  homeSpan.setStatusPin(GreenLED);          
  homeSpan.setStatusAutoOff(300); 
```

and when i comment out the line `homeSpan.setControlPin(ControlButton); ` Everything works fine. 
i asked ChatGPT and it told me the following:

> The current wiring of the button and the configuration in the code could contradict how HomeSpan expects the control pin. HomeSpan usually expects a LOW to HIGH transition on the defined control pin. Since the button is connected to GPIO 27 with an external pull-down resistor (10 kOhm), the pin is kept LOW by default and goes HIGH when the button is pressed (since it is connected to 3.3V). (Auto translated from German to English)

Any coding solution? Cant rewire the button. I already soldered it to the Pulldown resistance.

3.3V --- Pushbutton --- GPIO 27 (ESP32)
                                    |
                                    10 kOhm (Pull-Down)
                                    |
                                   GND

**Comment by felix6262** [comment]
And if I've already opened an issue anyway, I have a second question: Is there a way, the code will work although there is no wireless connection. I programmed a ventilation. A Sensor measures the humidity and when a value is reached, the fan starts and later stops independently. Therefor is no connection need to wifi and Apple Home. But right now nothing works, until i set up the Wifi and apple home.
